configure

```diff
@@ -5997,7 +5997,6 @@ EOF
 	cc_check $DISCORD_CFLAGS $DISCORD_LIBS -ldiscord-rpc && _discord=yes
```

Comment:
I would think that all this test is now useless.
Do we still need the discord_rpc.h header file to build?

Reply:
We still need discord_rpc.h for some status structure but it's easy to copy the relevant structure to our file. Should I?

Reply:
I would say yes.
That simplifies build.